grails-app/views/customer/index.gsp

```diff
@@ -6,9 +6,18 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Cadastro de cliente</title>
+        <asset:stylesheet src="errornotify.css"/>
```

Comment:
se esse css for usado para todas as mensagens de erro, recomendo importar no application.css